PokeAlarm/Manager.py

```diff
@@ -360,7 +331,7 @@ def process_pokemon(self, pkmn):
                 log.debug("Filter dist was not checked because the manager has no location set.")
 
             # Check the CP of the Pokemon
-            if cp != '?':
+            if cp is not None and cp != '?' and cp != 'unkn':
```

Comment:
I don't think it is appropriate to check if variables == None here. I touch base on this more in webhook structs.